modules/aggs-matrix-stats/src/main/java/org/elasticsearch/search/aggregations/support/ArrayValuesSourceAggregationBuilder.java

```diff
@@ -260,76 +207,6 @@ protected abstract ArrayValuesSourceAggregatorFactory innerBuild(QueryShardConte
                                                                      AggregatorFactory parent,
                                                                      AggregatorFactories.Builder subFactoriesBuilder) throws IOException;
 
-    public ValuesSourceConfig config(QueryShardContext queryShardContext, String field, Script script) {
```

Comment:
Huh, was this all just copy pasta from the main VSConfig resolve?

Reply:
Not exactly.  There are differences, but I don't think they matter.  For example, this code ignores the script and just treats a query that specified a script as if it was unmapped.  Which, honestly, seems more like a bug waiting to happen than anything else.  (The parser already enforces not sending a script, so I'm not worried that going to suddenly start getting script errors)

Reply:
:+1: